src/Controller/FigureFormController.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class FigureFormController extends AbstractController
+{
+    /**
+     * @Route("/form/figure", name="create_figure")
```

Comment:
Naming can be improved and rationalized :)

Reply:
Done :)